vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -108,10 +108,15 @@ installBpftrace() {
 }
 
 disableNtpAndTimesyncdInstallChrony() {
-    # Disable systemd-timesyncd
-    systemctl_stop 20 30 120 systemd-timesyncd || exit $ERR_STOP_OR_DISABLE_SYSTEMD_TIMESYNCD_TIMEOUT
-    systemctl disable systemd-timesyncd || exit $ERR_STOP_OR_DISABLE_SYSTEMD_TIMESYNCD_TIMEOUT
-
+    # Disable systemd-timesyncd if present
+    status=$(systemctl show -p SubState --value systemd-timesyncd)
+    if [ $status == 'dead' ]; then
```

Comment:
we already do this for ntp, see L120-127

Reply:
we should probably factor this out to some common func we can call whenever we want to mess with a service we _think_ is running, im more than fine with this for now though